Orange/widgets/visualize/utils/widget.py

```diff
@@ -704,10 +704,17 @@ def _manual_move_finish(self, anchor_idx, x, y):
     def _get_projection_data(self):
         if self.data is None or self.projection is None:
             return None
+
+        names = get_unique_names(self.data.domain, [a.name for a in
```

Comment:
Maybe the variables should be copied only when necessary, i.e. when the obtained names are different than the proposed.